dashboard/modules/metrics/export/grafana/dashboards/default_grafana_dashboard.json

```diff
@@ -1234,6 +1244,7 @@
         "defaults": {},
         "overrides": []
       },
+      "description": "A total number of active failed, and pending nodes from the cluster. \n\nACTIVE: A node is alive and available.\n\nFAILED: A node is dead and not available. The node is considered dead when the raylet process on the node is terminated.\n\nPending: A node is being started by the Ray cluster launcher. The node is unavailable now because it is being provisioned and initialized. The node will get into the failed state if it cannot be provided (e.g., there's no available node from the cloud provider) or failed to setup (e.g., setup_commands have errors). ",
```

Comment:
```suggestion
      "description": "A total number of active failed, and pending nodes from the cluster. \n\nACTIVE: A node is alive and available.\n\nFAILED: A node is dead and not available. The node is considered dead when the raylet process on the node is terminated. The node will get into the failed state if it cannot be provided (e.g., there's no available node from the cloud provider) or failed to setup (e.g., setup_commands have errors). \n\nPending: A node is being started by the Ray cluster launcher. The node is unavailable now because it is being provisioned and initialized.",
```